packages/modal/src/components/drawer/Drawer.tsx

```diff
@@ -16,31 +40,27 @@ export interface DrawerProps
     | "style"
     | "parentSelector"
   > {
-  width?: string;
   background?: string;
   zIndex?: number;
   onRequestClose?: () => void;
-  /**
-   * Which direction the drawer will appear from.
-   * @default left
-   * @param {String('left'|'right')}
-   */
-  slideFrom?: SlideFrom;
   /**
    * Portal target, HTML element. If not set, portal is not used.
    */
   portalTarget?: HTMLElement | null;
 }
 
 export const Drawer: React.FC<DrawerProps> = ({
-  width = "370px",
   background,
   zIndex,
   children,
   slideFrom = "left",
   portalTarget,
   ...reactModalProps
 }) => {
+  const height =
+    "height" in reactModalProps ? reactModalProps.height : undefined;
+  const width = "width" in reactModalProps ? reactModalProps.width : undefined;
```

Comment:
I don't like this

Reply:
Proposal?

Reply:
Please be more descriptive in comments :)